extensions/panache/mongodb-panache/deployment/src/main/java/io/quarkus/mongodb/panache/deployment/PanacheResourceProcessor.java

```diff
@@ -336,8 +347,13 @@ void buildMutiny(CombinedIndexBuildItem index,
             if (modelClasses.add(classInfo.name().toString()))
                 modelEnhancer.collectFields(classInfo);
         }
+
+        // iterate over all the entity classes
```

Comment:
do we need these comments?

Reply:
I copied them for the implementation of the impertive version.
this method is really big and complex so the comments help me to maintain it yes ...

Reply:
I personally do not see if they add any value.. upon arriving here the loop tells me that an iteration is going on..same for the one below about reflection registration.

Reply:
While I also don't think this particular comment adds much value, I think it's way overkill to ask people to put less comments. In general comments are good and should be encouraged.